### PR TITLE
Ensure we use python 3 everywhere [ci full]

### PR DESCRIPTION
**Pull request by eoger**
The support for Python 2 will end in 2020, let's ensure we use python 3 to avoid surprises.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ commands:
             pushd ..
             git clone https://chromium.googlesource.com/external/gyp.git
             pushd gyp
-            python setup.py install
+            python3 setup.py install
```
Comment (vladikoff) [inline .circleci/config.yml]: in https://github.com/mogemimi/pomdog/wiki/How-to-Install-GYP i saw "Python 2.7+ (GYP does not support Python 3, at the moment)" , I guess that's old?

Comment (eoger) [reply .circleci/config.yml]: Yeah this was actually fixed a few months ago.

Comment (eoger) [reply .circleci/config.yml]: https://chromium-review.googlesource.com/c/external/gyp/+/1639564

Comment (vladikoff) [reply .circleci/config.yml]: nice!

**Pull request merged**